src/parser/calculate_size.ts

```diff
@@ -81,6 +81,8 @@ function calculateElement(
         return (name != null ? Buffer.byteLength(name, 'utf8') + 1 : 0) + 1;
       } else if (value['_bsontype'] === 'ObjectId' || value['_bsontype'] === 'ObjectID') {
         return (name != null ? Buffer.byteLength(name, 'utf8') + 1 : 0) + (12 + 1);
+      } else if (value['_bsontype'] === 'UUID') {
```

Comment:
its a general question to just make sure you have noticed it. consider no action since its out of scope of the PR

Reply:
It's not really a part of my changes, but it seems like objects keys are used when `calculateElement` are called, and a stringified index in (this) case of arrays (we're talking about [this line](https://github.com/steffenagger/js-bson/blob/28355231f0aaaa9d671785e32e45df22bdcfed15/src/parser/calculate_size.ts#L17) right?).

Reply:
yep that one. I wonder why `toString` instead of calling the function with the number since there seems to be a case for `number` in `calculateElement`
https://github.com/steffenagger/js-bson/blob/28355231f0aaaa9d671785e32e45df22bdcfed15/src/parser/calculate_size.ts#L57

That's just a side note. maybe worth pointing this out to the maintainers

Reply:
Hmm, but that's for the `value`. The `i.toString()` is the `name` (it's just using the array-index as "key").

Reply:
```
. . . . . .. . . . . . . . . . . ,.-‘”. . . . . . . . . .“~.,
. . . . . . . .. . . . . .,.-”. . . . . . . . . . . . . . . . . .“-.,
. . . . .. . . . . . ..,/. . . . . . . . . . . . . . . . . . . . . . . ”:,
. . . . . . . .. .,?. . . . . . . . . . . . . . . . . . . . . . . . . . .\,
. . . . . . . . . /. . . . . . . . . . . . . . . . . . . . . . . . . . . . ,}
. . . . . . . . ./. . . . . . . . . . . . . . . . . . . . . . . . . . ,:`^`.}
. . . . . . . ./. . . . . . . . . . . . . . . . . . . . . . . . . ,:”. . . ./
. . . . . . .?. . . __. . . . . . . . . . . . . . . . . . . . :`. . . ./
. . . . . . . /__.(. . .“~-,_. . . . . . . . . . . . . . ,:`. . . .. ./
. . . . . . /(_. . ”~,_. . . ..“~,_. . . . . . . . . .,:`. . . . _/
. . . .. .{.._$;_. . .”=,_. . . .“-,_. . . ,.-~-,}, .~”; /. .. .}
. . .. . .((. . .*~_. . . .”=-._. . .“;,,./`. . /” . . . ./. .. ../
. . . .. . .\`~,. . ..“~.,. . . . . . . . . ..`. . .}. . . . . . ../
. . . . . .(. ..`=-,,. . . .`. . . . . . . . . . . ..(. . . ;_,,-”
. . . . . ../.`~,. . ..`-.. . . . . . . . . . . . . . ..\. . /\
. . . . . . \`~.*-,. . . . . . . . . . . . . . . . . ..|,./…..\,__
,,_. . . . . }.>-._\. . . . . . . . . . . . . . . . . .|. . . . . . ..`=~-,
. .. `=~-,_\_. . . `\,. . . . . . . . . . . . . . . . .\
. . . . . . . . . .`=~-,,.\,. . . . . . . . . . . . . . . .\
. . . . . . . . . . . . . . . . `:,, . . . . . . . . . . . . . `\. . . . . . ..__
. . . . . . . . . . . . . . . . . . .`=-,. . . . . . . . . .,%`>–==“
. . . . . . . . . . . . . . . . . . . . _\. . . . . ._,-%. . . ..`\.
```

Reply:
ignore